src/pages/content/components/twitter/tweetHandlers.tsx

```diff
@@ -84,36 +84,25 @@ export function handleOpTweet(tweet: HTMLElement) {
   tweet.setAttribute("data-dl-tweet-check", "true");
 }
 
-
-function checkForSolAddress(tweetText: string) {
-  const regexUpper = /[A-Z]/;
-  const regexLower = /[a-z]/;
-  const regexNumber = /[0-9]/;
-
-  const solanaAddressRegex = /([1-9A-HJ-NP-Za-km-z]{39,44})/g;
-  const res = tweetText.match(solanaAddressRegex);
-  if (!res) return false;
-  return res.some((str) => regexUpper.test(str) && regexLower.test(str) && regexNumber.test(str));
```

Comment:
Filtering out address without characters in any of these 3 categories isn't right, because it would result in false negatives - it is possible to have Solana address without numbers or lower/upper cased letter.